zim/plugins/tableeditor.py

```diff
@@ -506,6 +507,8 @@ def create_treeview(self, model):
 			cell = Gtk.CellRendererText()
 			tview_column = Gtk.TreeViewColumn(headcol, cell)
 			tview_column.set_sizing(Gtk.TreeViewColumnSizing.AUTOSIZE)  # allow column shrinks
+			tview_column.set_resizable(True);
+			tview_column.set_reorderable(True);
```

Comment:
Reordering and resizing are lost on page reload. Remove the semicolons for consistency; they're not needed